src/vivarium_census_prl_synth_pop/components/immigration.py

```diff
@@ -133,21 +135,42 @@ def on_time_step(self, event: Event):
             event.index,
         )
 
+        num_households_immigrating = self._round_stochastically(
```

Comment:
Why are GQ and individual immigration extracted to separate methods while household is not?

Reply:
Only because they had more in common with each other. I've extracted household immigration to a method, which I agree is nicer.